include/zim/search_iterator.h

```diff
@@ -46,7 +46,8 @@ class search_iterator : public std::iterator<std::bidirectional_iterator_tag, En
         search_iterator& operator--();
         search_iterator operator--(int);
 
-        std::string get_url() const;
+        std::string get_path() const;
+        std::string get_dbData() const;
```

Comment:
Make this private, we don't want to allow a user to get the dbData.

As we are using it in test we need to have it.
But you can do as for `Archive` for some methods (https://github.com/openzim/libzim/blob/master/include/zim/archive.h#L401-L405)

Reply:
This is a very neat solution! :+1: